src/util-dpdk-rte-flow.c

```diff
@@ -0,0 +1,931 @@
+/* Copyright (C) 2025 Open Information Security Foundation
+ *
+ * You can copy, redistribute or modify this Program under the terms of
+ * the GNU General Public License version 2 as published by the Free
+ * Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * version 2 along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+ * 02110-1301, USA.
+ */
+
+/**
+ *  \defgroup dpdk DPDK rte_flow rules util functions
+ *
+ *  @{
+ */
+
+/**
+ * \file
+ *
+ * \author Adam Kiripolsky <adam.kiripolsky@cesnet.cz>
+ *
+ * DPDK rte_flow rules util functions
+ *
+ */
+
+#include "decode.h"
+#include "flow-bypass.h"
+#include "flow-hash.h"
+#include "flow-storage.h"
+#include "runmode-dpdk.h"
+#include "util-debug.h"
+#include "util-dpdk.h"
+#include "util-dpdk-ice.h"
+#include "util-dpdk-rte-flow.h"
+#include "util-dpdk-rte-flow-pattern.h"
+#include "util-device-private.h"
+#include "runmodes.h"
+#include "tm-threads.h"
+#include "suricata.h"
+
+#ifdef HAVE_DPDK
+#include <rte_ring.h>
+
+#if RTE_VERSION >= RTE_VERSION_NUM(21, 0, 0, 0)
+
+#define INITIAL_RTE_FLOW_RULE_COUNT_CAPACITY 5
+#define DATA_BUFFER_SIZE                     1024
+#define COUNT_ACTION_ID                      128
```

Comment:
I think it is is little weird to set it to 128 specifically, is there any reason to do that, and not 0, 1 or something else?